Makefile

```diff
@@ -348,8 +351,10 @@ endif
 cleanup-tests:
 ifneq ($(RESTY_IMAGE_BASE),src)
 	docker-compose -f test/kong-tests-compose.yaml down
+	docker-compose -f test/kong-tests-compose.yaml rm -f
+	docker-compose -f docker-compose.yml down
```

Comment:
Is `docker-compose.yml` used anywhere? Not sure this down and rm belong here in this target.

Reply:
It is used and there is a fair bit of duplication between the two docker-compose files but I'll tackle that in a different PR. I tried merging them in this one and it was very challenging